packages/lib/services/synchronizer/syncInfoUtils.test.ts

```diff
@@ -200,6 +200,89 @@ describe('syncInfoUtils', () => {
+					'hasBeenUsed': true,
+				},
+			],
+			'ppk': {
+				'value': {
+					'id': 'SNQ5ZCs61KDVUW2qqqqHd3',
+					'keySize': 2048,
+					'privateKey': {
+						'encryptionMethod': 4,
+						'ciphertext': '{"iv":"Z2y11b4nCYvpmQ9gvxELug==","v":1,"iter":10000,"ks":256,"ts":64,"mode":"ccm","adata":"","cipher":"aes","salt":"0dqWvU/PUVQ=","ct":"8CvjYayXMpLsrAMwtu18liRfewKfZVpRlC0D0I2FYziyFhRf4Cjqi2+Uy8kIC8au7oBSBUnNU6jd04ooNozneKv2MzkhbGlXo3izxqCMVHboqa2vkPWbBAxGlvUYQUg213xG61FjZ19ZJdpti+AQy7qpQU7/h5kyC0iJ2aXG5TIGcBuDq3lbGAVfG/RlL/abMKLYb8KouFYAJe+0bUajUXa1KJsey+eD+2hFVc+nAxKOLe1UoZysB77Lq43DRTBFGH2gTSC1zOXxuZeSbFPPN0Cj+FvV7D5pF9LhUSLPDsIiRwF/q+E506YgDjirSZAvW1Y2EEM22F2Mh0I0pbVPFXhhBafqPLRwXmUGULCnA64gkGwctK5mEs985VVSrpQ0nMvf/drg2vUQrJ3exgl43ddVSOCjeJuF7F06IBL5FQ34iAujsOheRNvlWtG9xm008Vc19NxvhtzIl1RO7XLXrrTBzbFHDrcHjda/xNWNEKwU/LZrH0xPgwEcwBmLItvy/NojI/JKNeck8R431QWooFb7cTplO4qsgCQNL9MJ9avpmNSXJAUQx8VnifKVbzcY4T7X7TmJWSrpvBWV8MLfi3TOF4kahR75vg47kCrMbthFMw5bvrjvMmGOtyKxheqbS5IlSnSSz5x7wIVz0g3vzMbcbb5rF5MuzNhU97wNiz3L1Aonjmnu8r3vCyXTB/4GSiwYH7KfixwYM68T4crqJ0VneNy+owznCdJQXnG4cmjxek1wmJMEmurQ1JtANNv/m43gzoqd62V6Dq05vLJF+n7CS9HgJ3FTqYVCZLGGYrSilIYnEjhdaBpkcnFrCitbfYj+IpNC6eN6qg2hpGAbmKId7RLOGwJyda0jkuNP9mTqWOF+6eYn8Q+Y3YIY"}',
+					},
+					'publicKey': '-----BEGIN RSA PUBLIC KEY-----\nMIIBCgKCAQEAiSTY5wBscae/WmU3PfVP5FYQiuTi5V7BjPcge/6pXvgF3zwe43uy\nTWdzO2YgK/a8f3H507clcGlZN4e0e1jZ/rh4lMfaN\nugfNo0RAvuwn8Yniqfb69reygJywbFBIauxbBpVKbc21MLuCbPkVFjKG7qGNYdF4\nc17mQ8nQsbFPZcuvxsZvgvvbza1q0rqVETdDUClyIrY8plAjMgTKCRwq2gafP6eX\nWpkENAyIbOFxSKXjWy0yFidvZfYLz4mIRwIDAQAB\n-----END RSA PUBLIC KEY-----',
+					'createdTime': 1633274368892,
+				},
+				'updatedTime': 1633274368892,
+			},
+			'appMinVersion': '0.0.0',
+		};
+
+		const syncInfo = new SyncInfo();
+		syncInfo.load(JSON.stringify(initialData));
+
+		const filteredSyncInfo = syncInfo.filterSyncInfo();
+
+		expect(filteredSyncInfo).toEqual({
+			'activeMasterKeyId_': {
+				'updatedTime': 0,
+				'value': '400227d2222c4d3bb7346514861c643b',
+			},
+			'appMinVersion_': '0.0.0',
+			'e2ee_': {
+				'updatedTime': 0,
+				'value': true,
+			},
+			'masterKeys_': [
+				{
+					'created_time': 1515008161362,
+					'encryption_method': 4,
+					'hasBeenUsed': true,
+					'id': '400227d8a77c4d3bb7346514861c643b',
+					'source_application': 'net.cozic.joplin-desktop',
+					'updated_time': 1708103706234,
+				},
+			],
+			'ppk_': {
+				'updatedTime': 1633274368892,
+				'value': {
+					'createdTime': 1633274368892,
+					'id': 'SNQ5ZCs61KDVUW2qqqqHd3',
+					'keySize': 2048,
+					'privateKey': {
+						'ciphertext': '{"iv":"Z2y11b4nCYvpm...TqWOF+6eYn8Q+Y3YIY"}',
+						'encryptionMethod': 4,
+					},
+					'publicKey': '-----BEGIN RSA PUBLI... RSA PUBLIC KEY-----',
+				},
+			},
+			'version_': 3,
+		});
+	});
```

Comment:
Ok but that's not good, is it? "version" becomes "version_", "ppk" becomes "ppk_" etc.

Reply:
fixed